.ci/release/Jenkinsfile

```diff
@@ -84,14 +84,19 @@ pipeline {
             }
           }
         }
-        stage('Check master build status') {
-        when { expression { params.check_branch_ci_status } }
-         steps {
-             // If this build is not green: https://apm-ci.elastic.co/job/apm-agent-java/job/apm-agent-java-mbp/job/master/
-             whenTrue(!buildStatus(host: 'apm-ci.elastic.co', job: ['apm-agent-java', 'apm-agent-java-mbp', 'master'], return_boolean: true)) {
-               input(message: "WARNING! The master build is not passing. Do you wish to continue?")
-             }
-         }
+        stage('Check build status') {
+          when {
+            allOf {
+              expression { params.check_branch_ci_status }
+              not { branch 'stable' }
+            }
```

Comment:
There is no need to validate the `stable`. Besides, stable builds don't exist as a Jenkins Multibranch Pipeline since they are filtered in https://github.com/elastic/apm-agent-java/blob/c203cd11f18e15f576bd98be67488f0f1a279250/.ci/jobs/apm-agent-java-mbp.yml#L16